libs/pika/execution_base/include/pika/execution_base/any_sender.hpp

```diff
@@ -447,7 +447,7 @@ namespace pika::execution::experimental::detail {
             new (p) any_receiver_impl(PIKA_MOVE(receiver));
         }
 
-        void set_value(Ts... ts) && override
+        void set_value(Ts&&... ts) && override
         {
             pika::execution::experimental::set_value(PIKA_MOVE(receiver), PIKA_MOVE(ts)...);
```

Comment:
I think this should be:
```suggestion
            pika::execution::experimental::set_value(PIKA_MOVE(receiver), PIKA_FORWARD(Ts, ts)...);
```
_if_ we support `any_sender` with reference types. However, for now, we could instead simply static assert that it's only used with non-reference types since I think we don't have an immediate use case for storing reference types. It may be needed in the future though so you could open an issue for that.